memCheck/topProcess.py

```diff
@@ -0,0 +1,26 @@
+def topProcess(numOfProcesses):
+    import psutil
+
+    ad_pids = []
+    procs = []
+    for p in psutil.process_iter():
+        with p.oneshot():
+            try:
+                mem = p.memory_full_info()
+                info = p.as_dict(attrs=['name', 'memory_info'])
+            except psutil.AccessDenied:
+                ad_pids.append(p.pid)
+            except psutil.NoSuchProcess:
```

Comment:
Pass on zombie pids.  The shorthand call is supposed to handle this for us but if we switch to the internals we should do this.